R/read_pvolfile.R

```diff
@@ -127,9 +128,9 @@ read_pvolfile_body <- function(file, param = c(
     }
   } else {
     pvol_tmp <- tempfile()
-
+    rlang::check_installed("vol2birdR", "to read `NEXRAD` files")
```

Comment:
This will only work when the package is on CRAN - currently I get a message
```
> read_pvolfile("~/Desktop/KDIX20201005_020122_V06")
ℹ The package "vol2birdR" is required to read `NEXRAD` files
✖ Would you like to install it?

1: Yes
2: No

Selection: 1
Error in loadNamespace(x) : there is no package called ‘vol2birdR’
In addition: Warning message:
package ‘vol2birdR’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages 
```
This message doesn't explain to the user where to obtain vol2birdR

Reply:
`rlang::check_installed` has `reason` and `action` arguments, those can potentially be used to give a more informative message, and automatically install from github

Reply:
All the tests that require vol2birdR currently fail if vol2birdR is not installed, which would be the default. How do we make sure these tests are run on CRAN, i.e. how do we force installation of vol2birdR on CRAN test servers?

Reply:
If `vol2birdR` is released within the next week should we bother fixing this for this short time assuming not many people will work with the bleeding edge package?

Reply:
Since we will also have to change it back then in a weeks time?